spec/features/admin/dashboard_spec.rb

```diff
@@ -21,20 +21,19 @@
     customer_4 = create(:customer, first_name: 'Zach', last_name: "Doe")
     customer_5 = create(:customer, first_name: 'Charlie', last_name: "Rey")
 
-    merchant_1 = create(:merchant_with_invoices, invoice_count: 6, customer: customer_1, invoice_status: 2)
-    merchant_2 = create(:merchant_with_invoices, invoice_count: 3, customer: customer_2, invoice_status: 2)
-    merchant_3 = create(:merchant_with_invoices, invoice_count: 8, customer: customer_3, invoice_status: 2)
-    merchant_4 = create(:merchant_with_invoices, invoice_count: 1, customer: customer_4, invoice_status: 2)
-    merchant_5 = create(:merchant_with_invoices, invoice_count: 4, customer: customer_5, invoice_status: 2)
-
+    merchant_1 = create(:merchant_with_transactions, transaction_count: 6, customer: customer_1, transaction_result: 0)
```

Comment:
This is only creating one customer per merchant. Is this what you want? Do we need to create a second customer for one of the merchants?

Reply:
I just wanted to have different merchants. it's not required for the user story.